packages/suite/src/views/settings/device/index.tsx

```diff
@@ -199,13 +217,32 @@ const Settings = () => {
                     <TextColumn
                         title={<Translation id="TR_FIRMWARE_VERSION" />}
                         description={
-                            <>
+                            <Version>
                                 <Translation
                                     id="TR_YOUR_CURRENT_FIRMWARE"
-                                    values={{ version: getFwVersion(device) }}
+                                    values={{
+                                        version: (
+                                            <VersionTooltip content={getFwRevision(device) || ''}>
+                                                <VersionLink
```

Comment:
we do have this special component for external links, it handles the cases when TOR is enabled.
you can get inspiration from here:
https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/components/firmware/Initial.tsx#L148-L152

Reply:
That special component is not used for displaying suite version. Will likely want to change that at some point I would think.

https://github.com/trezor/trezor-suite/blob/9e8c13a5b2b666a2a06c7703fd3aa8e43dde67f9/packages/suite/src/views/settings/index.tsx#L362

Reply:
As mentioned in the other comment, `TrezorLink` was designed for trezor.io links.